Sources/Publish/API/Website.swift

```diff
@@ -62,6 +62,7 @@ public extension Website {
     /// - parameter deploymentMethod: How to deploy the website.
     /// - parameter additionalSteps: Any additional steps to add to the publishing
     ///   pipeline. Will be executed right before the HTML generation process begins.
+    /// - parameter plugins: Plugins to be installed beforehand.
```

Comment:
Very nitpicky, but perhaps specifying "beforehand" a bit more to say "at the start of the publishing process"?